### DISCUSSION TITLE
How control ownership of netbox-sync job in changelog ?

### DISCUSSION THREAD

**Discussion by jancovic** [bb-Ricardo/netbox-sync#284] [General]
I realized that after netbox-sync job is executed, In changelog is showing updates as executed per username defined in Netbox.
It is confusing.
How can I control it/change it to show it as some "service account job" ?

**Comment by bb-Ricardo** [comment]
Hi,

You need to create a service account in NetBox and create an API token for this user. And then set this API token in the configuration for `netbox.api_token`

**Comment by jancovic** [comment]
Thanks, is working! 
One more question: Is there some option how to somehow remove this vcenter logs from changelog ?
Because looks that it is filling  out ( daily rescan ) changelog... too many logs and then is not very clear,  what real changes were made by users. Users changes are lost there.

**Reply by bb-Ricardo** [reply]
Hi, this you have to ask the NetBox people. I don't know how to do that in NetBox, Sorry.

**Reply by jancovic** [reply]
ok, no problem

**Comment by jancovic** [comment]
resolved
